docs/integrators/deep-history-squad.md

```diff
@@ -35,143 +39,339 @@ In the example above, the key `726573657276650000000a55544b2d326638306539` is de
+        { File = "config.toml", Path = "StateTriesConfig.PeerStatePruningEnabled", Value = "true" },
+        { File = "config.toml", Path = "GeneralSettings.StartInEpochEnabled", Value = "false" }
+    ]
+```
+
+Above, the most important settings for retaining the non-pruned history are:
+
+```
+StateTriesConfig.AccountsStatePruningEnabled = false
+StoragePruning.AccountsTrieCleanOldEpochsData = false
+```
 
 :::tip
-As of October 2022, a public repository with non-pruned databases for both _mainnet_ and _devnet_ is under construction. This repository would take the shape of a _Digital Ocean (S3-compatible) Space_. Once the repository is ready, the data can be downloaded via **[db-archive-scripts](https://github.com/multiversx/mx-chain-db-archive-scripts)** - documentation will follow.
+Apart from the configuration above, the setup of a deep-history observer is identical to a regular full-history observer.
+:::
+
+:::warning
+Never attach a non-pruned database to a regular observer (i.e. that does not have the above settings) - unless you are not interested into the deep-history features. The regular observer irremediably removes, trucates and prunes the data (as configured, for storage efficiency).
 :::
 
 [comment]: # (mx-context-auto)
 
-### Reconstructing non-pruned database
+## Downloading non-pruned database
 
-An alternative to downloading a non-pruned history is to reconstruct it locally (on your own infrastructure).
+Archives supporting historical lookup are available to download from a Google Cloud Storage bucket.
 
-Under the hood, the reconstruction process relies on the **[import-db](https://docs.multiversx.com/validators/import-db/)** feature, which allows us to reprocess previously processed blocks - and, while doing so, for our purposes, we'll also retain the whole, non-pruned history. For our purposes, the `import-db` procedure requires a **target database** (placed in the folder `node-workdir/db`) and a **source database** (usually placed in the folder `node-workdir/import-db/db`).
+:::important
+Documentation in this section is preliminary and subject to change.
+:::
 
-It follows that, in order to reconstruct the history for an observer, we need (to download) **two database archives**: an _old archive_ and a _new archive_. For reconstructing the history of a whole squad, `4 x 2` archives are required (to be downloaded).
+[comment]: # (mx-context-auto)
 
-_Downloading_ the necessary archives and _unarchiving_ them is encapsulated in a step called **reconstruction bootstrapping**.
+## Reconstructing non-pruned databases
 
-[comment]: # (mx-context-auto)
+An alternative to downloading the non-pruned history is to reconstruct it locally (on your own infrastructure). The reconstruction process is based on the **[import-db](/validators/import-db/)** feature, which re-processes past blocks - and, while doing so, retains the whole, non-pruned accounts history.
```

Comment:
Added a short note. But now I think that would be easier for users from the community to use a regular sync instead of `import-db`. In a next PR, we can:

a) drop the details regarding reconstruction using `import-db`, and only describe the simpler approach (even if it takes a bit more time, depending on the availability of peers, plus the network traffic). 

b) have the regular-sync approach described side-by-side with the `import-db` approach.

Which one should we pick?

Reply:
I would choose b.